src/app/admin/index.ts

```diff
@@ -0,0 +1,17 @@
+import { checkedToUsername } from "@domain/users"
+import { AccountsRepository } from "@services/mongoose"
+
+export const getUserByUsername = async function (username: string) {
+  const usernameValid = checkedToUsername(username)
+  if (usernameValid instanceof Error) throw usernameValid
+
+  const username_ = username as Username
+
+  const accounts = AccountsRepository()
+  return accounts.findByUsername(username_)
+}
+
+export const getUserByPhone = async function (phone: string) {
+  const accounts = AccountsRepository()
+  return accounts.findByPhone(phone as PhoneNumber)
```

Comment:
phone must not be part of account, also avoid this type of direct cast (you can implement the same strategy used in `getUserByUsername` using `checkedToUsername` to validate input)